markdown/developers/tutorials/dsl2_modules_tutorial.md

```diff
@@ -292,17 +301,27 @@ You should first open `tests/modules/fgbio/fastqtobam/main.nf` and create a shor
 > :soon: this example is using available test data, chosen for Sarek functionalities. It will be updated according to the new [scheme](https://github.com/nf-core/modules/blob/master/tests/config/test_data.config)
 
 In our test workflow we have to define the two mandatory inputs.
-We know the test data is using QIAseq library preparation, and therefore we use the following read structure string
+We know the test data is using QIAseq library preparation, and therefore we need to specify read structure string as an optional input, via the `args` variable.
+
+For this we update the `nextflow.config` to pass the string to the module.
 
 ```nextflow
-params.read_structure = "+T 12M11S+T"
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
```

Comment:
I would worry someone might delete it thinking it's not temporary, even thoughit's from the template... unless I should just add a comment to that effect?

Reply:
Maybe just highlight that the `ext.args` thing is what the user should be looking at? It might take a beginner a moment to work out what they should be looking at is all.

Reply:
See suggestion above

Reply:
Yep, looks good